drivers/syscon/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 
 zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_SYSCON_GENERIC          syscon.c)
+zephyr_library_sources_ifdef(CONFIG_EFUSE_BFLB              efuse_bflb.c)
```

Comment:
@nordicjm ,

Should we keep `sysconn_vendor_foo` parter for naming,  `sysconn_bflb_efuse.c`?
Possibly this would imply on Kconfig symbol naming?

Reply:
done

Reply:
The pattern that I expect to see is

<driver>[_<vendor>][_<family/series>][_<ip_core>].c

sysconn_bflb_efuse.c

Reply:
First, it's syscon, not sysconn, secondly, please look at the files, it is done:
![image](https://github.com/user-attachments/assets/d96b36b5-64cf-429f-ad45-e3fba1afd0e3)
![image](https://github.com/user-attachments/assets/fd1f48d8-af2e-465f-8cd5-0c98b085461c)

Reply:
> Should we keep sysconn_vendor_foo parter for naming, sysconn_bflb_efuse.c?
Possibly this would imply on Kconfig symbol naming?

Would be good, can see it's been done